### PR TITLE
config.toml: change "Ack" to Support; "Ack" is an expression connotin…

### PR DESCRIPTION
**Pull request by ajpowelsnl**
Change "Ack" to Support in webpage heading

### CODE REVIEW

File: config/_default/config.toml [hunk 1]
```diff
@@ -199,9 +199,9 @@ weight = 4
 
 [[languages.en.menu.main]]
 identifier = "acknowledgments"
-name = "Ack"
+name = "Support"
 url = "/acknowledgments/"
-weight = 70
+weight = 50
```
Comment (dalg24) [inline config/_default/config.toml:204]: Why did you update the weight?  What does this do?

File: config/_default/config.toml [hunk 2]
```diff
@@ -199,9 +199,9 @@ weight = 4
 
 [[languages.en.menu.main]]
 identifier = "acknowledgments"
-name = "Ack"
+name = "Support"
```
Comment (dalg24) [inline config/_default/config.toml:202]: Why Support and not Acknowledgements?

File: config/_default/config.toml (hunk 1 above)
Comment (dalg24) [reply config/_default/config.toml:204]: If you don't know what it does then don't change it

File: config/_default/config.toml (hunk 2 above)
Comment (dalg24) [reply config/_default/config.toml:202]: Is support not potentially confusing if one is looking for user-support?

Comment (ajpowelsnl) [reply config/_default/config.toml:202]: For clarity, "Sponsors" is probably the better choice.   I can fix up the body of the text to make it non-redundant and clear.  Also, there are other sponsors, and I'll say something like, "Sponsors include ..."

File: config/_default/config.toml (hunk 1 above)
Comment (ajpowelsnl) [reply config/_default/config.toml:204]: Well, it's an educated guess, but it does make sense to try the original setting before changing.  If we end up with the crowded headings, then we can revisit.

File: config/_default/config.toml (hunk 2 above)
Comment (dalg24) [reply config/_default/config.toml:202]: Sure change it to Sponsors then

File: config/_default/config.toml (hunk 1 above)
Comment (dalg24) [reply config/_default/config.toml:204]: As far as I can tell it controls the order in which they appear, not the size of the font.

**Review by dalg24 — COMMENTED** [review]
If you want to propose the renaming of the tab link without reworking the acknowledgments section I will merge.

The update to the `acknowledgments` section needs to be looked more carefully.

File: config/_default/config.toml (hunk 1 above)
Comment (ajpowelsnl) [reply config/_default/config.toml]: After looking at it, I agree that "weight" controls the position on the page.

**Comment by ajpowelsnl** [comment]
> If you want to propose the renaming of the tab link without reworking the acknowledgments section I will merge.
> 
> The update to the `acknowledgments` section needs to be looked more carefully.

I submit a second PR.

**Comment by crtrott** [comment]
What about moving the Sponsors thing into the About category, instead of having its separate top level menu entry?

**Review by dalg24 — APPROVED** [review]

**Comment by ajpowelsnl** [comment]
That makes sense.  I will outline a wider set of streamlining changes, and pass it by you

**Pull request merged**